source/BcmI2cController.h

```diff
@@ -265,8 +265,8 @@ class BcmI2cControllerClass : public I2cControllerClass
     const ULONG _DIV_USED_MASK = 0x0000FFFF;  // Mask of non-reserved bits in the Clock Divider Register
 
     // Standard (100khz) and high (400khz) divider values.
-    const ULONG CDIV_100KHZ = 1500;
-    const ULONG CDIV_400KHZ = 376;      // Actually 398.9khz, divider value must be even
+    const ULONG CDIV_100KHZ = 2500;
```

Comment:
Yes, this is needed for the speedup.
No, we're not removing the 400KHz support. It's updated (below) as well to account for the new clock speed.